src/lcd.h

```diff
@@ -8,13 +8,18 @@
 #ifndef LCD_H_
 #define LCD_H_
 
-#define FB_HEIGHT (64)
-#define FB_WIDTH (128)
+#define FB_HEIGHT 		(64)
+#define FB_WIDTH 		(128)
```

Comment:
Whitespace changes probably shouldn't be pushed back, unless they are part of a cleanup patch.